clusters/app.ci/supplemental-ci-images/html-proofer_bc.yaml

```
@@ -0,0 +1,35 @@
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+metadata:
+  name: html-proofer
+  namespace: ci
+spec:
+  tags:
+    - name: "3.19.0"
+      from:
+        kind: DockerImage
+        name: "docker.io/klakegg/html-proofer:3.19.0"
+---
+kind: BuildConfig
+apiVersion: build.openshift.io/v1
+metadata:
+  namespace: ci
+  name: html-proofer-with-bash
+spec:
+  successfulBuildsHistoryLimit: 1
+  failedBuildsHistoryLimit: 2
+  output:
+    to:
+      kind: ImageStreamTag
```

```suggestion
      kind: DockerImage
```

Did it work out with `ImageStreamTag` in your test when you push to your own repo?